packages/opentelemetry-scope-async-hooks/README.md

```diff
@@ -1,4 +1,26 @@
-OpenTelemetry AsyncHooks ContextManager
-======================================================
+# OpenTelemetry AsyncHooks-based Scope Manager
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
 
-TODO
+This package provides [async-hooks][async-hooks-doc] based scope manager which is used internally by OpenTelemetry plugins to propagate specific scope between function calls and async operations. It only target NodeJS since async-hooks is only available there.
```

Comment:
```suggestion
This package provides [async-hooks][async-hooks-doc] based scope manager which is used internally by OpenTelemetry plugins to propagate specific scope between function calls and async operations. It only targets Node.js since async-hooks is only available there.
```